lib/mini_twitter_client.rb

```diff
@@ -0,0 +1,81 @@
+require 'faraday'
+require 'json'
+
+class Tweet
+  def initialize(name = 'name', email = 'email@email.com', message = 'message', id = nil)
+    @name = name
+    @email = email
+    @message = message
+    @id =id
+  end
+  def name
```

Comment:
Stylistic:

```suggestion

  def name
```

Add empty lines between methods ;)